pkgs/development/tools/out-of-tree/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchgit, qemu, docker }:
+
+buildGoPackage rec {
+  name = "out-of-tree-${version}";
+  version = "1.0.1";
+  rev = "v${version}";
+
+  buildInputs = [ qemu docker ];
+
+  goPackagePath = "code.dumpstack.io/tools/out-of-tree";
+
+  src = fetchgit {
+    inherit rev;
```

Comment:
I'm pretty `rev` need to be like `refs/tags/${version}` for `fetchgit`.
Could you also move the attribute into here instead of inheriting it?

Reply:
fixed